spec/acceptance/basic_candlepin_spec.rb

```diff
@@ -9,7 +9,25 @@
     it { is_expected.to be_listening }
   end
 
-  describe command('curl -k -s -o /dev/null -w \'%{http_code}\' https://localhost:8443/candlepin/status') do
-      its(:stdout) { should eq "200" }
+  describe command("curl -k -s -o /dev/null -w '%{http_code}' https://localhost:8443/candlepin/status") do
+    its(:stdout) { should eq "200" }
+  end
+
```

Comment:
It's perhaps arguable that this could be removed, since the TLS enabled tests below also confirm that the candlepin status API is responding as expected

However I think keeping it around for granularity makes sense, as it allows us to more quickly diagnose whether or not future test failures are TLS related